tests/clients/fake_dapr_server.py

```diff
@@ -74,6 +76,21 @@ def PublishEvent(self, request, context):
         context.set_trailing_metadata(trailers)
         return empty_pb2.Empty()
 
+    def SaveState(self, request, context):
+        headers = ()
+        trailers = ()
+        for state in request.states:
+            state = json.loads(MessageToJson(state))
+            self.store[state['key']] = state['value'].decode('utf-8')
```

Comment:
we have helpers to convert str to bytes or vice versa. - https://github.com/dapr/python-sdk/blob/c592371ba38f056487be519ccf8d6c1a42a75737/dapr/clients/grpc/_helpers.py#L55-L72